FHEM/00_SIGNALduino.pm

```diff
@@ -2535,7 +2535,7 @@ SIGNALduino_Parse_MC($$$$@)
 		   	SIGNALduino_Log3 $name, 5, "$name: extracted data $bitData (bin)";
 		   	
 		   	my $method = lib::SD_Protocols::getProperty($id,"method");
-		    if (!exists &$method)
+		    if (!exists &$method || !defined &{ $method })
 			{
 				SIGNALduino_Log3 $name, 5, "$name: Error: Unknown function=$method. Please define it in file $0";
 			} else {
```

Comment:
Ich muss hier nochmal einhaken:
Wenn ich das richtig verstehe, wird bei falscher method oder bei nicht existierender method eine Fehlermeldung nur bei verbose 5 ausgegeben und es erfolgt keine weitere Auswertung. Der User bekommt im Normalfall (verbsoe 3) davon nichts mit.

Warum drehst du das Ganze nicht einfach rum:
Bei existierender method diese anwenden, sonst halt keine anwenden und einfach die Auswertung fortsetzen.

Reply:
Existieren wird der Eintrag method immer, solange nicht jemand "rumbastelt".

Die Fehlermeldung, ob die definierte Methode auch existiert ist eher für Entwickler gedacht, welche die Angabe der Methode ja vornehmen.

Reply:
Man könnte den Eintrag aber auch einfach weglassen, wenn man keine Vorverarbeitung braucht, da es sowieso in einem selbst entwickelten Modul weiter verarbeitet wird.

Reply:
Die "Vorverarbeitung" soll prüfen, 
1)  ob die maximale Länge überschritten wurde 
2)  Die Daten von einer binären darstellung in eine hexadezimale wandeln

Ich verstehe nicht, wieso dieses flexiblen Auswerten geändert werden soll.

Reply:
Ich dachte ja nur, das es ein guter Zeitpunkt wäre, das gleich zu optimieren...